### DISCUSSION TITLE
Adding a new track

### DISCUSSION THREAD

**Discussion by parteekcoder** [aiortc/aiortc#1106] [Q&A]
From the client I am sending a message in datachannel that `send new stream`. Then in the server I want to send next track to the client.

But it is failing, i have tried both replaceTrackk and addTrack , none of them worked.

**Comment by el07694** [comment]
This will no work 
Maybe a renegotiation is required

**Comment by el07694** [comment]
Do you want to implement something like live radio - live tv?

Then you can use pyav and pydub to create one MediaStreamTrack which will have all the stream data.
